media-sound/mpdscribble/mpdscribble-0.23.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson ninja-utils
+
+DESCRIPTION="An MPD client that submits information to Audioscrobbler"
+HOMEPAGE="https://www.musicpd.org/clients/mpdscribble/"
+SRC_URI="https://www.musicpd.org/download/${PN}/${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+RDEPEND=">=media-libs/libmpdclient-2.5
+	>=dev-libs/boost-1.62
+	dev-libs/libgcrypt
+	>=net-misc/curl-7.18"
```

Comment:
Dependencies are alphabetically ordered, please fix

Reply:
You can also change them in:

dev-libs/boost
dev-libs/libgcrypt
media-libs/libmpdclient
net-misc/curl

as these blockers doesn't exist anymore